app/editor/src/features/admin/notifications/dashboard/utils/formatDate.ts

```diff
@@ -0,0 +1,17 @@
+import moment from 'moment';
+/**
+ * @param {string | Date} date - The date to format
+ * @param {boolean} includeTime - Whether to include the time in the formatted date
+ * @returns {string} - The formatted date
+ * @description - This function takes a date and returns a formatted date string
+ */
+export const formatDate = (date: string | Date | undefined, includeTime?: boolean): string => {
```

Comment:
Seeing duplicating with subscriber and editor code.  Ideally we'd move this to `tno-core` and update each app to use the shared identical method.